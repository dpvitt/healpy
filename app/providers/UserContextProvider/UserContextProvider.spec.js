import React from 'react';
import { shallow } from 'enzyme';
import { UserContextProvider } from './UserContextProvider';

describe('<UserContextProvider />', () => {
  const render = shallow(
    <UserContextProvider>
      <p>child</p>
    </UserContextProvider>,
  );

  it('should render', () => {
    expect(render).toMatchSnapshot();
  });
});
