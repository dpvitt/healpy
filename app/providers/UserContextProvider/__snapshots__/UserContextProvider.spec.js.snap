// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserContextProvider /> should render 1`] = `
<ContextProvider
  value={
    Array [
      null,
      [Function],
    ]
  }
>
  <p>
    child
  </p>
</ContextProvider>
`;
